tomcat/tomcat-jta/src/test/resources/context.xml

```diff
@@ -8,15 +8,21 @@
               factory="org.jboss.narayana.tomcat.jta.TransactionSynchronizationRegistryFactory"/>
 
     <!-- Database resources -->
+    <!-- Don't forget to change the xaDataSource in transactionalDataSource to match if you uncomment this
     <Resource name="h2DataSource" uniqueName="h2Datasource" auth="Container" type="org.h2.jdbcx.JdbcDataSource" username="sa"
-              user="sa" password="sa" url="jdbc:h2:mem:testdb" description="H2 Data Source" loginTimeout="0"
-              factory="org.h2.jdbcx.JdbcDataSourceFactory"/>
+                  user="sa" password="sa" url="jdbc:h2:mem:testdb" description="H2 Data Source" loginTimeout="0"
+                  factory="org.h2.jdbcx.JdbcDataSourceFactory"/>
+    -->
+    <Resource name="psqlDataSource" uniqueName="psqlDataSource" auth="Container" type="org.postgresql.xa.PGXADataSource"
```

Comment:
I wonder if we should not provide H2 as default option? And use a profile in `pom.xml` which would use different `context.xml` file for CI tests? My point is that's hard to run integration tests on local machine (especially without VPN) without manual changes in `context.xml` (while necessary changes are not described anywhere).

Reply:
Good point - let's do it that way. I will tweak to make that change.

Reply:
I have made that change now. You can run it with -Dpostgres-db=true or similar, per default it will run with H2